docs/content/developer/stardust/addresses.mdx

```diff
@@ -16,7 +16,6 @@ It's possible to convert a legacy Bech32 address back to hex format by doing the
 
 1. Take the part after `iota1`. Using the address from the above paragraph as an example, iota1**qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xqgyzyx**.
 2. Base32 decode it. 
```

Comment:
It's not as simple as that, it's not the usual base32 charset

Reply:
True, I changed the whole section and also included an example using an existing library, makes no sense to explain the bech32 specification here